src/__tests__/CreateMeal.test.js

```diff
@@ -0,0 +1,10 @@
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import CreateMeal from "../components/CreateMeal";
+
+describe("CreateMeal", () => {
+  test("renders create meal component", async () => {
+    const allIngridients = ["ground beef", "pasta", "meat"];
+    render(<CreateMeal allIngridients={allIngridients} />);
```

Comment:
The only thing you test here is that there are no exceptions in render. Is it what you want? If "yes" then change name to make it more obvious.